elastic/beats
# Rename s3 input to aws-s3 input

## What does this PR do?

This PR is to rename Filebeat `s3` input to `aws-s3` input.
Closes https://github.com/elastic/integrations/issues/488

## Checklist




## How to test this PR locally



1. Test the Filebeat `aws-s3` input:
```
filebeat.inputs:
- type: aws-s3
  enabled: true
  queue_url: https://sqs.eu-central-1.amazonaws.com/428152502467/filebeat-aws-elb-test
  credential_profile_name: elastic-beats
```
This should start `aws-s3` input with no error.

2. Test Filebeat filesets that use `aws-s3` input:
```
./filebeat modules enable aws
```
and change `modules.d/aws.yml` to enable a fileset in `aws` module. For example:
```
- module: aws
  cloudtrail:
    enabled: true
    var.queue_url: https://sqs.us-east-1.amazonaws.com/428152502467/test-fb-ks
    var.credential_profile_name: elastic-beats
```
Then run Filebeat and you should see no error. If there are new logs going into the S3 which generates new SQS messages in `test-fb-ks`, you should also see the events in Elasticsearch.
